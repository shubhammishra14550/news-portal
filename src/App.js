import logo from './logo.svg';
import './App.css';
import Navigation from './componenets/Navigation';

function App() {
  return (
    <div className="App">
      <Navigation/>
      <header className="App-header">
        <p>
          Work in progress for <code>News Bulletin</code>.
        </p>
        <hr/><hr/>
        <img src={logo} className="App-logo" alt="logo" />
      </header>
    </div>
  );
}

export default App;
