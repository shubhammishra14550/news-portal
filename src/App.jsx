import logo from './logo.svg';
import './App.css';
import Navigation from './componenets/navigation/Navigation';

export default function App() {
  return (
    <div className="App">
      <Navigation/>
      <header className="App-header">
        <p>
          Work in progress for <code>News Bulletin &hearts;</code>.
        </p>
        <hr/><hr/>
        <img src={logo} className="App-logo" alt="logo" />
      </header>
    </div>
  );
}
