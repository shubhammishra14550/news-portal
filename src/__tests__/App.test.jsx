import { render } from '@testing-library/react';
import App from '../App';

describe('App tests', () => {
  it('should render correctly', () => {
    const { asFragment } = render(<App />);
    expect(asFragment()).toMatchSnapshot();
  });
});
