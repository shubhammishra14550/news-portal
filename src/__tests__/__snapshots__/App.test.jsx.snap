// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App tests should render correctly 1`] = `
<DocumentFragment>
  <div
    class="App"
  >
    <nav
      class="navbar navbar-expand navbar-dark bg-dark"
    >
      <div
        class="container"
      >
        <a
          class="navbar-brand"
          href="#bulletin"
        >
          News Express
        </a>
        <div
          class="me-auto navbar-nav"
        >
          <a
            class="nav-link"
            data-rr-ui-event-key="#bulletin"
            href="#bulletin"
          >
            Bulletin
          </a>
          <a
            class="nav-link"
            data-rr-ui-event-key="#addNews"
            href="#addNews"
          >
            Add News
          </a>
          <a
            class="nav-link"
            data-rr-ui-event-key="#filterNews"
            href="#filterNews"
          >
            Filter News
          </a>
        </div>
      </div>
    </nav>
    <header
      class="App-header"
    >
      <p>
        Work in progress for 
        <code>
          News Bulletin â™¥
        </code>
        .
      </p>
      <hr />
      <hr />
      <img
        alt="logo"
        class="App-logo"
        src="test-file-stub"
      />
    </header>
  </div>
</DocumentFragment>
`;
